python/nn4k/nn4k/executor/huggingface/__init__.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2023 OpenSPG Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied.
```

Comment:
Consider importing commonly used base classes here to avoid deep module nesting when importing from outside.
e.g.  `from nn4k.executor.huggingface.base.hf_args import HFModelArgs` is too long.
`from nn4k.executor.huggingface import HFModelArgs` may be better.

Reply:
OK, I have shorten both the base executor and args.